### DISCUSSION TITLE
Twitter Tweet Scraper

### DISCUSSION THREAD

**Discussion by leadsteelcopper** [AAndyProgram/SCrawler#93] [Ideas]
Hello,

Is there any plans to add the ability to copy tweets themselves? If not do you know of any other programs that can do that?

**Comment by AAndyProgram** [comment]
If by "themselves" you mean the text of the tweet, then no. If you mean something else, please describe in more detail.

**Reply by leadsteelcopper** [reply]
Yes that was what I meant. Do you know of any open source projects that have a good user interface like yours that can collect the entire tweet? Your program has an excellent interface that does not need the command line and is can get all the big mainstream sites.

**Reply by AAndyProgram** [reply]
No, I don't know of projects like you described. I have never been interested in such things.
And thank you for such kind words about my program and for like on the site. 😊
